pkgs/development/libraries/dbus/default.nix

```diff
@@ -13,6 +13,10 @@
 , docbook_xml_dtd_44
 , docbook-xsl-nons
 , xmlto
+
+, libxcb ? null
+, libXau ? null
+, libXdmcp ? null
```

Comment:
doesn't callPackage pass these in anyways?

Reply:
How would it?